cloud-core/src/test/java/cloud/commandframework/arguments/standard/DurationArgumentSuggestionsTest.java

```diff
@@ -0,0 +1,65 @@
+package cloud.commandframework.arguments.standard;
+
+import cloud.commandframework.CommandManager;
+import cloud.commandframework.TestCommandSender;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static cloud.commandframework.util.TestUtils.createManager;
+
+public class DurationArgumentSuggestionsTest {
+
+    private static CommandManager<TestCommandSender> manager;
+
+    @BeforeAll
+    static void setupManager() {
+        manager = createManager();
+        manager.command(manager.commandBuilder("duration")
+                .argument(DurationArgument.of("duration")));
+    }
+
+
+    @Test
+    void testDurationSuggestions() {
+        final String input = "duration ";
+        final List<String> suggestions = manager.suggest(new TestCommandSender(), input);
+        Assertions.assertEquals(Arrays.asList("1", "2", "3", "4", "5", "6", "7", "8", "9"), suggestions);
+
+        final String input2 = "duration 1";
+        final List<String> suggestions2 = manager.suggest(new TestCommandSender(), input2);
+        Assertions.assertEquals(Arrays.asList("1d", "1h", "1m", "1s"), suggestions2);
+
+        final String input3 = "duration 1d";
+        final List<String> suggestions3 = manager.suggest(new TestCommandSender(), input3);
+        Assertions.assertTrue(suggestions3.containsAll(Arrays.asList("1d1h", "1d1m", "1d1s")));
```

Comment:
Idk, I feel like these are weird suggestions for the already typed (and valid) duration 1d (and 1d2h below), personally I feel like it should only suggest when `1d1` is typed with suggestions `1d1h`, `1d1m` and `1d1s`

minor thingee, but ic test cases also only test one digit, maybe add some for more digits too?

Reply:
Thank you!